### DISCUSSION TITLE
[Fall 2024] Week 6 Assignment

### DISCUSSION THREAD

**Discussion by jooohyunpark** [jooohyunpark/itp-canvas-for-coders#8] [Assignments]
Work on your final project and be prepared to present it. (5 mins PT + 3 mins QA/comments)

- Free format - any supporting materials are welcome: documentations, inspirations, drawings, code snippets, etc.
- Walk us through your idea, process, learnings, future plans etc.

**Comment by Joyceyyzheng** [comment]
Progress Update - Joyce & Isabel
[https://joycezheng.notion.site/Week-6-Progress-Update-11b4ee8df27e807abc5bf20451eeaf99?pvs=4]
